pyci/include/pyci.h

```diff
@@ -677,4 +680,89 @@ struct SparseOp final {
     void add_row(const SQuantOp &, const GenCIWfn &, const long, ulong *, long *, long *);
 };
 
+/* FanCI objective classes. */
+
+template<class Wfn>
+class Objective {
+public:
+    std::size_t nproj;
```

Comment:
Is there somewhere a bit more description about what this variables are? 
For example I don't know what the `nconn` one stands for. I saw it somewhere else and I thought it may mean number of connected determinants, but not sure.